hail/python/test/hail/methods/relatedness/test_pc_air.py

```diff
@@ -0,0 +1,72 @@
+import unittest
+
+import numpy as np
+
+import hail as hl
+from hail.methods.relatedness.pc_air import _partition_samples
+from test.hail.helpers import resource
+
+
+class Tests(unittest.TestCase):
+    def test_partition_samples(self):
+        plink_path = resource('fastlmmTest')
+        mt = hl.import_plink(
+            bed=f'{plink_path}.bed', bim=f'{plink_path}.bim', fam=f'{plink_path}.fam', reference_genome=None
+        )
+        table = _partition_samples(mt.GT, 0.05)
+        unrelated = set(table.filter(table.is_in_unrelated).s.collect())
+        related = set(table.filter(~table.is_in_unrelated).s.collect())
+        expected_related = {
```

Comment:
How did you construct the expected result?

Reply:
I created this by running the code and copying the output, so this unit test serves as a regression test. I will see if I can improve on this.